GoogleSignIn/Tests/Unit/GIDSignInTest.m

```diff
@@ -503,6 +503,39 @@ - (void)testRestorePreviousSignInWhenSignedOut {
   [_authState verify];
 }
 
+- (void)testNotRestorePreviousSignInWhenSignedOutAndCompletionIsNil {
+  [[[_authorization expect] andReturn:_authState] authState];
+  [[[_authState expect] andReturnValue:[NSNumber numberWithBool:NO]] isAuthorized];
+
+  [_signIn restorePreviousSignInWithCompletion:nil];
+
+  XCTAssertNil(_signIn.currentUser);
+}
+
+- (void)testRestorePreviousSignInWhenCompletionIsNil {
+  [[[_authorization expect] andReturn:_authState] authState];
+  [[[_authState expect] andReturnValue:[NSNumber numberWithBool:YES]] isAuthorized];
+
+  OIDTokenResponse *tokenResponse =
+  [OIDTokenResponse testInstanceWithIDToken:[OIDTokenResponse fatIDToken]
```

Comment:
Please fix the indentation to match the rest of the file. `[OIDTokenResponse ...` should be indented four spaces from the column where text begins on line 519.